Expand on links to starters (#483)
vaadin/docs: articles/guide/start/index.asciidoc

[.breakout]
=== Non-Spring Boot Starters
The applications created using {wizard-name} are based on Spring Boot.

> Starters for other stacks like JavaEE or plain servlet are available in GitHub (note though that Fusion can only work with a Spring Boot project).

Reviewer: ```suggestion
Vaadin Flow starters for other stacks, such as JavaEE or plain servlet, are available in GitHub.
Note that Fusion only works with a Spring Boot project.
```
